docs/sources/project/advanced-contributing.md

```diff
@@ -137,3 +137,18 @@ The following provides greater detail on the process:
 
 14. Acceptance and merge!
 
+### Why in the world we have to do this
```

Comment:
I'd probably do this:

## Design Proposal > ## About the Advanced process

Then, I'd  drop your addition _before_ the sentence beginning _The following_

---> I'd also suggest the following changes -->
Docker is a large project. Our core team gets a great many design proposals. Design proposal discussions can span days, weeks, and longer.  The number of comments can reach the 100s.  In that situation, following the discussion flow and the decisions reached is crucial.

Making a pull request with a design proposal simplifies this process:

* you can leave comments on specific design proposal line
* replies around line are easy to track
* as a proposal changes and is updated, pages reset as line items resolve
* Git maintains the entire history

While proposals in pull requests do not end up merged into a master repository, they provide a convenient tool for managing the design process.

------>

This gist illustrates the whole thing: https://gist.github.com/a412e32c14f453df7dbc.git